src/root/file.d

```diff
@@ -73,7 +73,7 @@ struct File
             size_t size;
             stat_t buf;
             ssize_t numread;
-            char* name = this.name.toChars();
+            const(char)* name = this.name.toChars();
```

Comment:
It'll all have to be scrutinized line by line anyway, because of 0 termination, rewrite to toString, etc.